scipy/optimize/tests/test_minpack.py

```diff
@@ -157,15 +157,16 @@ def test_func_can_raise(self):
         def func(*args):
             raise ValueError('I raised')
 
-        with assert_raises(ValueError, match='I raised'):
+        with assert_raises(ValueError, message='I raised'):
```

Comment:
Are you sure this is the right thing to do? According to the docs https://docs.pytest.org/en/latest/reference.html#pytest-raises (and my own experiments when I wrote these tests), ``message`` just changes the text shown upon assertion failure. My intention was to check that we are catching the same ValueError that I raised.

Reply:
Right, I should have corrected "matches=" to "match=" and not to "message=". Fixed.